src/charm.py

```diff
@@ -407,6 +408,19 @@ def set_secret(self, scope: str, key: str, value: Optional[str]) -> None:
         else:
             raise RuntimeError("Unknown secret scope.")
 
+    def install_exporter(self) -> None:
+        """Install JMX exporter."""
+        subprocess.run(
+            f"wget -O {self.kafka_config.default_config_path}/jmx-exporter.jar https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.17.2/jmx_prometheus_javaagent-0.17.2.jar",
+            shell=True,
+        )
```

Comment:
I'd rather use requests library if possible. subprocess.run should be avoided for security and vulnerability reason, whenever possible. Here, using request library should be possible